### ISSUE TITLE
Stopped working since v3.1

### ISSUE THREAD

**Issue by stuartcusackie** [manogi/nova-tiptap#74]
v3.1 and v3.2 cause Vue errors for me and make Nova unusable. v3.0 is fine.

The problem occurs when updating a resource.  I get multiple errors as shown below and this also breaks navigation.

I have tried removing other nova packages but I think I have narrowed it down to this package. I have also tried republishing all nova assets.

```
TypeError: Cannot read properties of null (reading 'style')
    at Qs (runtime-dom.esm-bundler.js:1388:6)
    at beforeUnmount (runtime-dom.esm-bundler.js:1384:5)
    at cn (runtime-core.esm-bundler.js:158:18)
    at un (runtime-core.esm-bundler.js:166:17)
    at fr (runtime-core.esm-bundler.js:1914:7)
    at $ (runtime-core.esm-bundler.js:6202:9)
    at Object.remove (runtime-core.esm-bundler.js:6592:11)
    at $ (runtime-core.esm-bundler.js:6205:20)
    at G (runtime-core.esm-bundler.js:6294:7)
    at $ (runtime-core.esm-bundler.js:6195:7)

TypeError: Cannot read properties of null (reading 'parentNode')
    at remove (runtime-dom.esm-bundler.js:13:26)
    at a (runtime-core.esm-bundler.js:6257:7)
    at V (runtime-core.esm-bundler.js:6271:7)
    at $ (runtime-core.esm-bundler.js:6226:9)
    at Object.remove (runtime-core.esm-bundler.js:6592:11)
    at $ (runtime-core.esm-bundler.js:6205:20)
    at G (runtime-core.esm-bundler.js:6294:7)
    at $ (runtime-core.esm-bundler.js:6195:7)
    at Y (runtime-core.esm-bundler.js:6314:7)
    at $ (runtime-core.esm-bundler.js:6215:9)
```

**Comment by stevelacey** [comment]
I also hit this, the bug was introduced in 3.0.7

**Comment by SchwartzN** [comment]
Same problem here

**Comment by sajaddp** [comment]
Same problem here

**Comment by manogi** [comment]
I have a fresh laravel, fresh Nova and fresh Nova TipTap install here where the error you report does not occur. So please provide more detailed info about your setup, your Nova TipTap settings etc.

**Comment by manogi** [comment]
Found it. It had something to do with Novas Modal component not playing nice with using custom Modals, and also occurred only when using the Image or Link button.

I made a quick fix in v3.2.1. The error should disappear with that version.
